content/docs/3.0/faq/faq.md

```diff
@@ -0,0 +1,166 @@
+
+## Can I use TiKV as a key-value store?
+
+Yes. For now, we have a stable [TiKV driver for Go](https://github.com/pingcap/tidb/tree/master/store/tikv). There are also drivers for [other languages in development](../reference/clients/introduction.md).
+
+## How does TiKV compare to NoSQL databases like Cassandra, HBase, or MongoDB?
+
+TiKV is as scalable as NoSQL databases like to advertise, while including features like externally consistent distributed transactions and good support for stateless query layers such as TiSpark (Spark), Titan (Redis), or TiDB (MySQL)
+
+## What is the recommended number of replicas in the TiKV cluster? Is it better to keep the minimum number for high availability?
+
+Use 3 replicas for testing. If you increase the number of replicas, the performance declines but it is more secure. Whether to configure more replicas depends on the specific business needs.
+
+## If a node is down, will the service be affected? How long?
+
+TiKV uses Raft to synchronize data among multiple replicas (by default 3 replicas for each Region). If one replica goes wrong, the other replicas can guarantee data safety. Based on the Raft protocol, if a single leader fails as the node goes down, a follower in another node is soon elected as the Region leader after a maximum of 2 * lease time (lease time is 10 seconds).
+
+## Is the Range of the Key data table divided before data access?
+
+No. It differs from the table splitting rules of MySQL. In TiKV, the table Range is dynamically split based on the size of Region.
+
+## How does Region split?
+
+Region is not divided in advance, but it follows a Region split mechanism. When the Region size exceeds the value of the `region-split-size` or `region-split-keys` parameters, split is triggered. After the split, the information is reported to PD.
+
+## What are the features of TiKV block cache?
+
+TiKV implements the Column Family (CF) feature of RocksDB. By default, the KV data is eventually stored in the 3 CFs (default, write and lock) within RocksDB.
+
+- The default CF stores real data and the corresponding parameter is in `[rocksdb.defaultcf]`. The write CF stores the data version information (MVCC) and index-related data, and the corresponding parameter is in `[rocksdb.writecf]`. The lock CF stores the lock information and the system uses the default parameter.
+- The Raft RocksDB instance stores Raft logs. The default CF mainly stores Raft logs and the corresponding parameter is in `[raftdb.defaultcf]`.
+- Each CF has an individual block-cache to cache data blocks and improve RocksDB read speed. The size of block-cache is controlled by the `block-cache-size` parameter. A larger value of the parameter means more hot data can be cached and is more favorable to read operation. At the same time, it consumes more system memory.
```

Comment:
I think now block cache of CFs are merged together.

Reply:
How about:

"CFs have a shared block-cache to cache data blocks and improve RocksDB read speed."

Reply:
LGTM